frontend/apps/app/components/SessionDetailPage/components/Output/components/Header/ExportDropdown.tsx

```diff
@@ -20,11 +20,61 @@ import styles from './ExportDropdown.module.css'
 
 type Props = {
   schema: Schema
+  artifactDoc?: string
 }
 
-export const ExportDropdown: FC<Props> = ({ schema }) => {
+const generateAIPrompt = (schema: Schema, artifactDoc: string): string => {
+  const schemaResult = schemaToDdl(schema)
+
+  return `# Database Schema Design
+
+${artifactDoc}
+
+## SQL Schema
+\`\`\`sql
+${schemaResult.ddl}\`\`\`
```

Comment:
This is supposed to be a migration code (diff).However, it is a bit tricky to take it out, so the following PR is used to deal with it.